storage/localstore/export.go

```diff
@@ -59,11 +60,20 @@ func (db *DB) Export(w io.Writer) (count int64, err error) {
 	}
 
 	err = db.retrievalDataIndex.Iterate(func(item shed.Item) (stop bool, err error) {
+
+		// To remember the pinned content across export and import..
+		// store the pin counter in the header
+		paxMap := make(map[string]string)
```

Comment:
1. what does `pax` refer to?
2. why do you need to persist the values as strings? persisting as other types does not serialize?

Reply:
1) PAX headers are used in tar format to store custom key-value. When someone exports the DB and  imports it.. i wanted to store the pin counter of that file somewhere in the tar itself.. I picked PAX header for that..

2) No. PAX header supports only string maps.

Reply:
Removed this for now.